storage/innobase/xtrabackup/doc/source/howtos/recipes_xbk_partition.html

```diff
@@ -0,0 +1,118 @@
+tables. 
+
+Creating the backup
+================================================================================
+
+There are three ways of specifying which part of the whole data will be backed
+up: regular expressions (:option: --include`), enumerating the
+tables in a file (:option:`--tables-file`) or providing a list of
+databases (:option:` --databases`).
+
+The regular expression provided to this option will be matched against the fully
+qualified database name and table name, in the form of
+``database-name.table-name``.
+
+For example, this operation takes a back up of the partition ``p4`` from the table ``name`` located in the database ``imdb``::
+
+  $ xtrabackup --tables=^imdb[.]name#p#p4 --backup 
+
+
+Note that this option is passed to :option:`xtrabackup --tables` and is matched
+against each table of each database, the directories of each database will be
+created even if they are empty.
+
+Preparing the backup
+================================================================================
+
+For preparing partial backups, the procedure is analogous to :doc:`restoring
+individual tables <..restoring_individual_tables>` : apply the
+logs and use :option:`xtrabackup --export`:
+
+.. code-block:: bash
+
+   $ xtrabackup --apply-log --export /mnt/backup/2012-08-28_10-29-09
+
+You may see warnings in the output about tables that do not exist. This is
+because |InnoDB|-based engines stores its data dictionary inside the tablespace
+files besides the :term:`.frm` files. |xtrabackup| removes the missing tables (those that haven't been selected in the partial
+backup) from the data dictionary in order to avoid future warnings or errors::
+
+  InnoDB: in InnoDB data dictionary has tablespace id 51,
+  InnoDB: but tablespace with that id or name does not exist. It will be removed from data dictionary.
+  120828 10:25:28  InnoDB: Waiting for the background threads to start
+  120828 10:25:29 Percona XtraDB (http://www.percona.com) 1.1.8-20.1 started; log sequence number 10098323731
+  xtrabackup: export option is specified.
+  xtrabackup: export metadata of table 'imdb/name#P#p4' to file `./imdb/name#P#p4.exp` (1 indexes)
+  xtrabackup:     name=PRIMARY, id.low=73, page=3
+
+You should also see the notification of the creation of a file needed for
+importing (:term:`.exp` file) for each table included in the partial backup::
+
+  xtrabackup: export option is specified.
+  xtrabackup: export metadata of table 'imdb/name#P#p4' to file `./imdb/name#P#p4.exp` (1 indexes)
+  xtrabackup:     name=PRIMARY, id.low=73, page=3
+
+
+Restoring from the backups
+================================================================================
+
+Restoring should be done by :doc:`importing the tables
+<restoring_individual_tables>` in the partial backup to the
+server.
+
+First step is to create new table in which data will be restored :: 
+
+::
+
+   mysql> CREATE TABLE `name_p4` (
```

Comment:
In some note section mention to generate cfg file PXB need first Partition i.e p0. because it has the metadata to generate cfg file 

information about cfg file is also in  https://dev.mysql.com/doc/refman/8.0/en/innodb-table-import.html

The .cfg files contain metadata that is used for schema verification when importing the tablespace. FLUSH TABLES ... FOR EXPORT can only be run on the table, not on individual table partitions

Reply:
Added note.